sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/aad/AadResourceServerWebSecurityConfigurerAdapter.java

```diff
@@ -22,10 +26,31 @@
 public abstract class AadResourceServerWebSecurityConfigurerAdapter extends WebSecurityConfigurerAdapter {
 
     @Autowired
-    AadResourceServerProperties properties;
+    private AadResourceServerProperties properties;
+
+    private Converter<Jwt, Collection<GrantedAuthority>> jwtGrantedAuthorityConverter;
 
     /**
-     * configure
+     * Creates a new instance with the default configuration.
+     */
+    public AadResourceServerWebSecurityConfigurerAdapter() {
+    }
+
+    /**
+     * Sets the Azure AD properties and custom granted authority converter to creates a new instance,
+     * the custom granted authority converter can be null.
+     *
+     * @param properties the Azure AD properties for Resource Server
+     * @param jwtGrantedAuthorityConverter the custom converter for JWT granted authority
+     */
+    public AadResourceServerWebSecurityConfigurerAdapter(AadResourceServerProperties properties,
+                                                         Converter<Jwt, Collection<GrantedAuthority>> jwtGrantedAuthorityConverter) {
+        this.properties = properties;
+        this.jwtGrantedAuthorityConverter = jwtGrantedAuthorityConverter;
```

Comment:
you're checking NotNull at here https://github.com/Azure/azure-sdk-for-java/pull/32335/files#diff-ce112d381fe58ed8f9480c52baed5571933351148ab89b75987aaf0dc8658214R94 but why not here?